core/util/paths.ts

```diff
@@ -9,8 +9,32 @@ import { IdeType, SerializedContinueConfig } from "../index.js";
 
 dotenv.config();
 
-const CONTINUE_GLOBAL_DIR =
-  process.env.CONTINUE_GLOBAL_DIR ?? path.join(os.homedir(), ".continue");
+function getContinueGlobalDir(): string {
+  // Use user-configured directory if set.
+  const envGlobalDir = process.env.CONTINUE_GLOBAL_DIR;
+  if (envGlobalDir) {
+    return envGlobalDir;
+  }
+
+  const defaultDotDir = path.join(os.homedir(), ".continue");
+
+  // Backwards compatibility: use ~/.continue if it already exists
+  if (fs.existsSync(defaultDotDir)) {
+    return defaultDotDir;
+  }
+
+  // on Linux, prefer XDG directories by default
+  // https://specifications.freedesktop.org/basedir-spec/latest/index.html
+  const xdgDataDir = process.env.XDG_DATA_DIR;
+  if (xdgDataDir) {
+    return path.join(xdgDataDir, "continue");
+  }
+
```

Comment:
Would you mind adding same for Windows?
```suggestion
  // on Windows, prefer ~\AppData\Local
  const localAppDataDir =process.env.LOCALAPPDATA;
  if (localAppDataDir ) {
    return path.join(localAppDataDir , "Continue");
  }
```